tests/unit/core/reducers/test_categories.js

```diff
@@ -24,21 +28,19 @@ describe(__filename, () => {
     expect(loading).toEqual(false);
   });
 
-  describe('CATEGORIES_FETCH', () => {
-    it('sets loading', () => {
-      const state = categories(
-        initialState,
-        categoriesFetch({ errorHandlerId: 'some-handler' }),
-      );
-      expect(state.categories).toEqual(null);
-      expect(state.loading).toEqual(true);
-    });
+  it('sets loading', () => {
```

Comment:
Should this be inside the `describe` block for `CATEGORIES_FETCH`?

Reply:
Yep